tests/testsuite/search.rs

```diff
@@ -115,7 +115,10 @@ fn not_update() {
     cargo_process("search postgres")
         .replace_crates_io(registry.index_url())
         .with_stdout_data(SEARCH_RESULTS)
-        .with_stderr_data("") // without "Updating ... index"
+        .with_stderr_data(str![[r#"
```

Comment:
Please update the commits so each one passes tests

Reply:
I have rebased them into one commit.